akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -422,7 +422,7 @@ should be replaced by:
 .. includecode:: code/docs/MigrationsJava.java#query-param
 
 SynchronousFileSource and SynchronousFileSink
-============================================
+=============================================
```

Comment:
Oh I didn't realise it actually breaks when not aligned. I always fix them because of my OCD anyway :wink: